pallets/automation-time/src/lib.rs

```diff
@@ -578,13 +585,13 @@ pub mod pallet {
 		/// - End in a whole hour
 		/// - Be in the future
 		/// - Not be more than MaxScheduleSeconds out
-		fn is_valid_time(scheduled_time: UnixTime) -> Result<(), Error<T>> {
+		fn is_valid_time(scheduled_time: UnixTime) -> Result<(), DispatchError> {
```

Comment:
same here as above for dispatch error

Reply:
https://github.com/OAK-Foundation/OAK-blockchain/pull/187#discussion_r915399384